File: Python/15.py
faturamento = 0
fatItem=0
printf = print
i=1


printf ("\n\nEste programa determina o faturamento de uma empresa!")
n = int(input("\n\nDigite a quantidade de produtos: "))

for i in range(1,n+1):
    preco = float(input(f"\n\nDigite o preço do produto {i}: "))
    fatItem=n*preco
    faturamento=faturamento+fatItem
    i=i+1

printf (f"\n\nO faturamento da empresa é de R${faturamento}!")

printf ("\n\nObrigado por utilizar nosso sistema!\n\n")

File: C/15.cpp
#include <stdio.h>
#include <conio.h>
#include <locale.h>
#include <stdlib.h>

int main (void) {
	system ("cls");
	setlocale (LC_ALL, "Portuguese");
	float preco, fatItem, faturamento, n, i;
	faturamento = 0;
	fatItem=0;
	printf ("\n\nEste programa determina o faturamento de uma empresa!");
	printf ("\n\nDigite a quantidade de produtos: ");
	scanf ("%f", &n);

	for(i=1;i<=n;i++){
		printf ("\n\nDigite o preço do produto %.0f: ",i);
		scanf ("%f", &preco);
		fatItem=n*preco;
		faturamento=faturamento+fatItem;
	}
	printf ("\n\nO faturamento da empresa é de R$%.2f!",faturamento);

	printf ("\n\nObrigado por utilizar nosso sistema!\n\n");
	system ("Pause");
	return 0;
}

